packages/components/src/components/navigation-item/navigation-item.scss

```
@@ -82,11 +53,22 @@
 	display: inline-flex;
 
 	border-radius: $default-card-border-radius;
-	padding: $db-spacing-fixed-xs $db-spacing-fixed-sm;
 	white-space: nowrap; // we don't want to break
 	text-align: center;
 	align-items: center; // Centering the content vertically and horizontally
 
+	&:not(:has(a)) {
+		padding: $db-spacing-fixed-xs $db-spacing-fixed-sm;
+	}
+
+	&:has(a) {
+		a {
+			padding: $db-spacing-fixed-xs $db-spacing-fixed-sm;
+			inline-size: 100%;
+			text-align: start;
+		}
+	}
```

```suggestion
	a {
		padding: $db-spacing-fixed-xs $db-spacing-fixed-sm;
		inline-size: 100%;
		text-align: start;
	}
```

wouldn't this be the same?